src/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.block-1 {
+  background-color: #f00;
+  flex-grow: 1;
```

Comment:
Don't repeat yourself. You may write `flex-grow: 1;` in your `.block` class and it will be applied to all blocks